Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs

```diff
@@ -1042,6 +1042,31 @@ public IEnumerator HandleVisibilityTest()
             Assert.IsTrue(rotationHandleAxisY.gameObject.activeSelf, "rotation handle y not active");
             Assert.IsTrue(rotationHandleAxisZ.gameObject.activeSelf, "rotation handle z not active");
 
+            // test disabling all rotation handles before activating the gameobject
+            // verifies bug https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8239
+            boundsControl.gameObject.SetActive(false);
+            yield return null;
+            rotationHandlesConfig.ShowRotationHandleForX = false;
+            rotationHandlesConfig.ShowRotationHandleForY = false;
+            rotationHandlesConfig.ShowRotationHandleForZ = false;
+            boundsControl.BoundsControlActivation = BoundsControlActivationType.ActivateOnStart;
+            boundsControl.gameObject.SetActive(true);
+            yield return null;
+            // refetch transforms
+            rigRoot = boundsControl.transform.Find("rigRoot").gameObject;
```

Comment:
```suggestion
            boundsControl.gameObject.SetActive(true);
            yield return null;
            
            // refetch transforms
            rigRoot = boundsControl.transform.Find("rigRoot").gameObject;
```
nit: spaces for consistency with the rest of the test